src/views/transaction/transaction.view.jsx

```diff
@@ -141,6 +143,18 @@ function Transaction({
 
   React.useEffect(() => onCleanup, [onCleanup]);
 
+  function finishTransaction() {
```

Comment:
We can't do this here. The idea of this task is to remove the `TransactionConfirmation` and `TransactionError` components. This code should be inside the functions that are currently executing the asynchronous tasks: `deposit`, `transfer`, `withdraw`, `forceExit` and `exit` in `transaction.thunks.js`. The idea is also to display the error next to the `Transaction failed` message.

Reply:
I've moved logic from view to thunks